src/modules/m_cloaking.cpp

```diff
@@ -309,15 +310,16 @@ class ModuleCloaking : public Module
 	Version GetVersion() CXX11_OVERRIDE
 	{
 		std::string testcloak = "broken";
+		irc::sockets::sockaddrs sa;
 		if (Hash)
 		{
 			switch (mode)
 			{
 				case MODE_HALF_CLOAK:
-					testcloak = prefix + SegmentCloak("*", 3, 8) + suffix;
+					testcloak = GenCloak(sa, "", testcloak + std::to_string(domainparts));
```

Comment:
Does this break linking compat with 2.0? I think that it might...

Also, std::to_string is a C++11 addition. Use ConvToStr instead.